### PR TITLE
Rd 5999 cypress

### PR DESCRIPTION
**Pull request by Blewin**
## Description

cypress tests for rd-5999, base will be changed to master after merging rd-5999
## Checklist

- [x] My code follows the [UI Code Style](https://cloudifysource.atlassian.net/l/c/x8fq902p).
- [x] I followed the [UI Testing Guidelines](https://cloudifysource.atlassian.net/l/cp/udCVfvrx) and verified that all tests/checks have passed.
- [x] I verified if that change requires any update in the [documentation](https://cloudifysource.atlassian.net/l/c/4XKtPocy).
- [x] I added proper labels to this PR.

## Tests

 

added e2e tests:

system: https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3898/

[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3898/)

## Documentation

will be added in a separate ticket

### CODE REVIEW

**Comment by qooban** [comment]
@Blewin What's the status of this PR? Let's either finish it this week or reopen RD-5999, so all we'll be aware of the actual state.

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 1]
```diff
@@ -16,12 +16,15 @@ describe('Deployments View widget', () => {
     const blueprintName = `${specPrefix}blueprint`;
     const deploymentId = `${specPrefix}deployment_id`;
     const deploymentName = `${specPrefix}deployment_name`;
+    const labelName = `${specPrefix}label_name`;
+    const labelVal = `${specPrefix}label_val`;
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:20]: More accurate with the resource naming convention would be:
```suggestion
    const labelKey = `${specPrefix}label_name`;
    const labelValue = `${specPrefix}label_val`;
```

WDYT?

File: test/cypress/integration/widgets/deployments_view_spec.ts [hunk 2]
```diff
@@ -123,6 +130,34 @@ describe('Deployments View widget', () => {
...
+            getDeploymentsViewTable().within(() => {
+                cy.contains(deploymentName);
+                cy.contains(labelName).should('not.exist');
+            });
+
+            cy.log('Show a label');
+            cy.editWidgetConfiguration(widgetId, () => {
+                widgetConfigurationHelpers.toggleLabelsDropdown();
+                widgetConfigurationHelpers.getLabelsDropdown().within(() => {
+                    cy.get('[role="listbox"]').contains(labelName).click();
```
Comment (qooban) [inline test/cypress/integration/widgets/deployments_view_spec.ts:149]: Optional:

I'm not 100% sure, but probably you can use here shorter version:
```suggestion
                    cy.contains('[role="listbox"]', labelName).click();
```

See https://docs.cypress.io/api/commands/contains#Selector for details.

**Comment by Blewin** [comment]
some system checks fail, is it acceptable @qooban ? they seem unrelated

**Comment by qooban** [comment]
@Blewin 

> some system checks fail, is it acceptable @qooban ? they seem unrelated

Seems unrelated. If you update branch with latest `master` you shouldn't get these errors. I think you can just re-check `deploymentsView` widget as I mentioned in [the previous comment](https://github.com/cloudify-cosmo/cloudify-stage/pull/2255#issuecomment-1323640798). So I suggest to:
1. Update branch with latest master
2. Run Stage UI system tests on your branch with `TEST_SPEC` parameter set to `test/cypress/integration/widgets/deployments_view_spec.ts`

**Comment by Blewin** [comment]
new build, who dis: [Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3908/)

**Comment by Blewin** [comment]
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3909/)

**Comment by Blewin** [comment]
[Build Status](https://jenkins.cloudify.co/view/UI%20Health%20View/job/Stage-UI-System-Test/3939/)

**Review by qooban — APPROVED** [review]

**Pull request merged**